app/javascript/components/payment-error-button.vue

```diff
@@ -116,7 +117,21 @@ export default {
       this.showCartOnDebtModal = false;
       this.showNameTextBox = false;
       this.showDebtFinalMessage = true;
+      const products = this.cartProductsToReqFormat();
+      console.log(this.cartProductsToReqFormat);
+      invoiceApi.createDebtProduct({
+        "debtor": this.message,
+        "products": products,
+       });
     },
+      cartProductsToReqFormat() {
```

Comment:
Expected indentation of 4 spaces but found 6                         indent

Reply:
no se si algo pasa con la indentación o para variar el mono se rayó con el linter...

Reply:
creo que efectivamente sobran 2 espacios